### PR TITLE
Anaconda: Add install instructions for anaconda

### PR DESCRIPTION
**Pull request by seisman**
Closes https://github.com/seismo-learn/software/issues/49

### CODE REVIEW

File: source/anaconda/index.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
+Anaconda
+=========
+
+Anaconda 是一个 用于科学计算的 Python 发行版，支持 Linux、macOS 和 Windows。
+其提供了方便使用的包管理器和环境管理器。
```
Comment (core-man) [inline source/anaconda/index.rst:5]:
```suggestion
Anaconda 是一个用于科学计算的 Python 发行版，支持 Linux、macOS 和 Windows。
其提供了方便使用的包管理器和环境管理器。
```

File: source/anaconda/index.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+**建议所有 Python用户使用 Anaconda 而非 Linux 或 macOS 系统自带的 Python。**
```
Comment (core-man) [inline source/anaconda/index.rst:7]:
```suggestion
**建议所有 Python 用户使用 Anaconda 而非 Linux 或 macOS 系统自带的 Python。**
```

File: source/anaconda/index.rst [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+使用
+----
+
+Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包并管理虚拟环境，其详细用法见
+`conda 官方文档 <https://docs.conda.io/projects/conda/en/latest/index.html>`__。
+此处仅介绍常用的命令。
+
+创建虚拟环境::
+
+   # 虚拟环境名为 seismo-learn，python版本为3.8
```
Comment (core-man) [inline source/anaconda/index.rst:58]:
```suggestion
   # 虚拟环境名为 seismo-learn，python 版本为 3.8
```

File: source/anaconda/index.rst [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+   Windows 用户直接双击安装包即可安装。
+
+   Linux 用户在终端执行如下命令以安装 Anaconda::
+
+      $ bash Anaconda3-2020.11-Linux-x86_64.sh -b
+
+   macOS 用户再终端执行如下命令以安装 Anaconda::
+
+      $ bash Anaconda3-2020.11-MacOSX-x86_64.sh -b
```
Comment (core-man) [inline source/anaconda/index.rst:35]: What's the meaning of `-b` option?

File: source/anaconda/index.rst [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+----
+
+Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包并管理虚拟环境，其详细用法见
+`conda 官方文档 <https://docs.conda.io/projects/conda/en/latest/index.html>`__。
+此处仅介绍常用的命令。
+
+创建虚拟环境::
+
+   # 虚拟环境名为 seismo-learn，python版本为3.8
+   conda create --name seismo-learn python=3.8
```
Comment (core-man) [inline source/anaconda/index.rst:59]: Do we want to create an environment with a special python version?

File: source/anaconda/index.rst (hunk 4 above)
Comment (seisman) [reply source/anaconda/index.rst:35]: batch mode. No stopping for user input

File: source/anaconda/index.rst (hunk 5 above)
Comment (seisman) [reply source/anaconda/index.rst:59]: OK to remove the Python version

File: source/anaconda/index.rst (hunk 4 above)
Comment (core-man) [reply source/anaconda/index.rst:35]: You mean all the options will use the default values?

File: source/anaconda/index.rst [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+----
+
+Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包并管理虚拟环境，其详细用法见
+`conda 官方文档 <https://docs.conda.io/projects/conda/en/latest/index.html>`__。
+此处仅介绍常用的命令。
+
+创建虚拟环境::
+
+   # 虚拟环境名为 seismo-learn，python 版本为 3.8
+   conda create --name seismo-learn python=3.8
```
Comment (core-man) [inline source/anaconda/index.rst:59]:
```suggestion
   # 虚拟环境名为 seismo-learn，初始 Python 版本与 base 环境相同
   conda create --name seismo-learn
```

File: source/anaconda/index.rst [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+   conda deactivate
+
+搜索模块::
+
+   conda search numpy
+
+安装模块::
+
+   conda install numpy
+
```
Comment (core-man) [inline source/anaconda/index.rst:77]:
```suggestion

更新模块::

   conda update numpy
   
```

File: source/anaconda/index.rst [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+也可以使用 Python 自带的工具 pip 安装模块::
+
+   pip install numpy
+
+加速 conda
```
Comment (core-man) [inline source/anaconda/index.rst:82]:
```suggestion
加速下载
```

File: source/anaconda/index.rst [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+      [Clang 10.0.0 ] :: Anaconda, Inc. on darwin
+      Type "help", "copyright", "credits" or "license" for more information.
+      >>>
+
+使用
+----
+
+Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包并管理虚拟环境，其详细用法见
+`conda 官方文档 <https://docs.conda.io/projects/conda/en/latest/index.html>`__。
+此处仅介绍常用的命令。
```
Comment (core-man) [inline source/anaconda/index.rst:54]:
```suggestion
Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包、管理虚拟环境，其详细用法见
`conda 官方文档 <https://docs.conda.io/projects/conda/en/latest/index.html>`__。
此外，也可以使用 Python 自带的工具 ``pip`` 来安装 Python 包，其详细用法见
`pip 官方文档 <https://pip.pypa.io/en/stable/>`。我们建议尽可能使用 ``conda`` 来安装 Python 包，
仅在 Anaconda 没有提供需要的程序包时才使用 ``pip`` 来安装。
以下仅介绍常用的命令。
```

File: source/anaconda/index.rst [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+
+搜索模块::
+
+   conda search numpy
+
+安装模块::
+
+   conda install numpy
+
+也可以使用 Python 自带的工具 pip 安装模块::
```
Comment (core-man) [inline source/anaconda/index.rst:78]:
```suggestion
使用 pip 安装模块::
```

File: source/anaconda/index.rst [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+
+   pip install numpy
+
+加速 conda
+----------
```
Comment (core-man) [inline source/anaconda/index.rst:83]:
```suggestion
加速下载
------------
```

File: source/anaconda/index.rst [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+      Type "help", "copyright", "credits" or "license" for more information.
+      >>>
+
+使用
+----
+
+Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包并管理虚拟环境，其详细用法见
+`conda 官方文档 <https://docs.conda.io/projects/conda/en/latest/index.html>`__。
+此处仅介绍常用的命令。
+
```
Comment (core-man) [inline source/anaconda/index.rst:55]: I find I have to update anaconda first before I can install some packages.

```suggestion

更新 Anaconda 和 conda::

   conda update anaconda
   conda update conda

```

Comment (core-man) [inline source/anaconda/index.rst:55]:
```suggestion

添加第三方源 conda-forge::
   conda config --add channels conda-forge
 
```

**Comment by core-man** [comment]
I find you have not added `$` before `conda` or `pip`.

File: source/anaconda/index.rst (hunk 4 above)
Comment (seisman) [reply source/anaconda/index.rst:35]: Yes

File: source/anaconda/index.rst [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+2. 安装 Anaconda
+
+   Windows 用户直接双击安装包即可安装。
+
+   Linux 用户在终端执行如下命令以安装 Anaconda::
+
+      $ bash Anaconda3-2020.11-Linux-x86_64.sh -b
+
+   macOS 用户再终端执行如下命令以安装 Anaconda::
```
Comment (core-man) [inline source/anaconda/index.rst:33]:
```suggestion
   macOS 用户在终端执行如下命令以安装 Anaconda::
```

File: source/anaconda/index.rst [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+      Type "help", "copyright", "credits" or "license" for more information.
+      >>>
+
+使用
+----
+
+Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包、管理虚拟环境，其详细用法见
+`conda 官方文档 <https://docs.conda.io/projects/conda/en/latest/index.html>`__。
+此外，也可以使用 Python 自带的工具 ``pip`` 来安装 Python 包，其详细用法见
+`pip 官方文档 <https://pip.pypa.io/en/stable/>`。我们建议尽可能使用 ``conda`` 来安装 Python 包，
```
Comment (core-man) [inline source/anaconda/index.rst:55]:
```suggestion
`pip 官方文档 <https://pip.pypa.io/en/stable/>`__。我们建议尽可能使用 ``conda`` 来安装 Python 包，
```

File: source/anaconda/index.rst [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+仅在 Anaconda 没有提供需要的程序包时才使用 ``pip`` 来安装。
+以下仅介绍常用的命令。
+
+更新 Anaconda 和 conda::
+
+   conda update anaconda
+   conda update conda
+
+添加第三方源 conda-forge::
+   conda config --add channels conda-forge
```
Comment (core-man) [inline source/anaconda/index.rst:65]:
```suggestion
添加第三方源 conda-forge::

   conda config --add channels conda-forge
```

File: source/anaconda/index.rst [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+   conda update numpy
+   
+使用 pip 安装模块::
+
+   pip install numpy
+
+加速下载
+------------
+
+使用 conda 或 pip 下载模块时，可能速度较慢，此时可考虑使用清华大学提供的 Anaconda 和 pypi
```
Comment (core-man) [inline source/anaconda/index.rst:101]:
```suggestion
在中国使用 conda 或 pip 下载模块时，可能速度较慢，此时可考虑使用清华大学提供的 Anaconda 和 pypi
```

**Comment by core-man** [comment]
I find you have not added `$` before `conda` or `pip`. Do we want to ignore `$` here?

**Comment by seisman** [comment]
> I find you have not added `$` before `conda` or `pip`. Do we want to ignore `$` here?

I think we should.

**Comment by core-man** [comment]
> > I find you have not added `$` before `conda` or `pip`. Do we want to ignore `$` here?
> 
> I think we should.

Should ignore or NOT? Then, how about the above commands in install and in other software documentation.

**Comment by seisman** [comment]
> > > I find you have not added `$` before `conda` or `pip`. Do we want to ignore `$` here?
> > 
> > 
> > I think we should.
> 
> Should ignore or NOT? Then, how about the above commands in install and in other software documentation.

I think we should always prefix shell commands with the prompt `$` so that it's clear that it's a shell command.

File: source/anaconda/index.rst [hunk 17]
```diff
@@ -0,0 +1,111 @@
...
+
+   终端输入 ``python``，输出中看到 **Anaconda, Inc.** 字样即代表安装完成::
+
+      $ python
+      Python 3.8.5 (default, Sep  4 2020, 02:22:02)
+      [Clang 10.0.0 ] :: Anaconda, Inc. on darwin
+      Type "help", "copyright", "credits" or "license" for more information.
+      >>>
+
+使用
```
Comment (core-man) [inline source/anaconda/index.rst:49]:
```suggestion
常用命令
```

**Comment by seisman** [comment]
I'll make the changes

**Comment by core-man** [comment]
I find the dash in Github web is a little different as in our local editor.

**Comment by seisman** [comment]
> I find the dash in Github web is a little different as in our local editor.

I have no idea what you mean. Please post some screenshots.

File: source/anaconda/index.rst [hunk 18]
```diff
@@ -0,0 +1,111 @@
...
+
+使用 pip 安装模块::
+
+   $ pip install numpy
+
+加速下载
+--------
+
+在中国使用 conda 或 pip 下载模块时，可能速度较慢，此时可考虑使用清华大学提供的 Anaconda 和 pypi
+镜像以实现加速。具体用法见:
```
Comment (core-man) [inline source/anaconda/index.rst:102]:
```suggestion
在中国使用 conda 或 pip 下载模块时，可能速度较慢，此时可考虑使用清华大学提供的 Anaconda 和 pypi
镜像以实现加速 (pypi 是 pip 默认的软件包下载源)。具体用法见:
```

File: source/anaconda/index.rst [hunk 19]
```diff
@@ -0,0 +1,111 @@
...
+`pip 官方文档 <https://pip.pypa.io/en/stable/>`__。我们建议尽可能使用 ``conda`` 来安装 Python 包，
+仅在 Anaconda 没有提供需要的程序包时才使用 ``pip`` 来安装。
+以下仅介绍常用的命令。
+
+更新 Anaconda 和 conda::
+
+   $ conda update anaconda
+   $ conda update conda
+
+添加第三方源 conda-forge::
```
Comment (core-man) [inline source/anaconda/index.rst:64]:
```suggestion
添加 conda 的第三方软件包源 conda-forge::
```

**Comment by core-man** [comment]
> > I find the dash in Github web is a little different as in our local editor.
> 
> I have no idea what you mean. Please post some screenshots.

dash-github

**Comment by seisman** [comment]
> > > I find the dash in Github web is a little different as in our local editor.
> > 
> > 
> > I have no idea what you mean. Please post some screenshots.
> 
> dash-github

Yes, most editors (except vim) have a similar problem.

**Review by core-man — APPROVED** [review]

**Pull request merged**